cmd/guacgql/cmd/root.go

```diff
@@ -40,6 +40,12 @@ var flags = struct {
 	nPass  string
 	nRealm string
 
+	// Needed only if using ent backend
```

Comment:
I think at some point (not in this PR, just commenting here to start a discussion) we might want to separate these flags and have them active/present only if the backend to which they apply is included.

Thoughts @lumjjb @pxp928 @jeffmendoza ?

Reply:
+1, otherwise we will end up with many unused flags that will cause confusion.

Reply:
yea i like the prefix backend address idea!

Reply:
@ivanvanderbyl did you want to create an issue to capture this needed change?